yb-voyager/src/issue/ddl.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Let's move this file to `queryissue` pkg as we had thought of doing? It makes sense for all the definitions to be in `queryissue` rather than in `issue` pkg

Reply:
I will have to move everything then `constants.go`, `issue.go` to `queryissue` , is that fine?

Reply:
You know what, let's not club this change with this PR. I will take it up later.